### PR TITLE
New single role field and custom display labels

### PR DESCRIPTION
**Pull request by nalingia**
Hi!
I'd like to contribute to this package proposing new custom field (RoleSelect) and changing two existing fields to permit custom label display on roles and permissions.

New field is intended to provide a standard way to manage single-role users scenario.

Hope this could be useful to others.

### CODE REVIEW

File: src/PermissionBooleanGroup.php [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 class PermissionBooleanGroup extends BooleanGroup
 {
-    public function __construct($name, $attribute = null, callable $resolveCallback = null)
+    public function __construct($name, $attribute = null, $labelAttribute = null, callable $resolveCallback = null)
```
Comment (vyuldashev) [inline src/PermissionBooleanGroup.php:14]: This change may break some applications. May be put `labelAttribute` after `resolveCallback`?

File: src/RoleBooleanGroup.php [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
 class RoleBooleanGroup extends BooleanGroup
 {
-    public function __construct($name, $attribute = null, callable $resolveCallback = null)
+    public function __construct($name, $attribute = null, $labelAttribute = null, callable $resolveCallback = null)
```
Comment (vyuldashev) [inline src/RoleBooleanGroup.php:14]: Same here

File: src/RoleSelect.php [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+namespace Vyuldashev\NovaPermission;
+
+use Illuminate\Support\Collection;
+use Laravel\Nova\Fields\Select;
+use Laravel\Nova\Http\Requests\NovaRequest;
+use Spatie\Permission\PermissionRegistrar;
+use Spatie\Permission\Traits\HasPermissions;
+
+class RoleSelect extends Select
```
Comment (vyuldashev) [inline src/RoleSelect.php:11]: Please, add docs to readme of when/why and how to use it

File: src/PermissionBooleanGroup.php (hunk 1 above)
Comment (nalingia) [reply src/PermissionBooleanGroup.php]: Sure!

File: src/RoleSelect.php (hunk 3 above)
Comment (nalingia) [reply src/RoleSelect.php:11]: Done! :)

**Comment by vyuldashev** [comment]
Good job!

**Pull request merged**